dev/manual_tests/test/card_collection_test.dart

```diff
@@ -9,33 +9,31 @@ import 'package:test/test.dart';
 import '../card_collection.dart' as card_collection;
```

Comment:
The first commit is the interesting one, and you can see its diff if you look at that commit directly.